### PR TITLE
8 revise section security considerations

### PR DESCRIPTION
**Pull request by fluppe2**
Tried to adress security considerations phrased in

https://mailarchive.ietf.org/arch/msg/spasm/3XiW2VUdPN2It_0ToyqHQssnfL0/

and stress the importance of "unusual" backup and restore management.

### CODE REVIEW

File: draft-gazdag-x509-hash-sigs.md [hunk 1]
```diff
@@ -16,6 +16,11 @@ venue:
   github: x509-hbs/draft-gazdag-x509-hash-sigs
 
 author:
+-
+    ins: K. Bashiri
+    name: Kaveh Bashiri
+    org: BSI
+    email: tbd@tbd.tbd
```
Comment (BashiriK) [inline draft-gazdag-x509-hash-sigs.md:23]:
```suggestion
    email: kaveh.bashiri.ietf@gmail.com
```

File: draft-gazdag-x509-hash-sigs.md [hunk 2]
```diff
@@ -666,57 +617,44 @@ This ASN.1 Module builds upon the conventions established in [RFC5911].
...
+the risk that an OTS key will be reused accidentally. This is a new requirement
+that most developers will not be familiar with and requires careful handling.
+
+Various strategies for a correct state management can be applied:
+
+- Implement a track record of all signatures generated by a key pair associated
+  to a stateful HBS instance. This track record may be stored outside the
+  device which is used to generate the signature. Check the track record to
+  prevent OTS key reuse before a new signature is released. Drop the new
+  signature and hit your PANIC button if you spot OTS key reuse.
```
Comment (csosto-pk) [inline draft-gazdag-x509-hash-sigs.md:637]: I suggest to also put in the state reservation mechanism too. It is explained in section 5 of [MCGREW].

Comment (fluppe2) [reply draft-gazdag-x509-hash-sigs.md:645]: Added to the list of strategies. See https://github.com/x509-hbs/draft-gazdag-x509-hash-sigs/pull/12/commits/610f419cd21d94a5eb8c0bd5dce90e5512d91289

**Review by exception42 — APPROVED** [review]
All changes seem fine

**Pull request merged**